gateway/node.go

```diff
@@ -21,12 +23,18 @@ import (
 	"github.com/filecoin-project/lotus/lib/sigs"
 	_ "github.com/filecoin-project/lotus/lib/sigs/bls"
 	_ "github.com/filecoin-project/lotus/lib/sigs/secp"
+	"github.com/filecoin-project/lotus/metrics"
 	"github.com/filecoin-project/lotus/node/impl/full"
 )
 
 const (
 	DefaultLookbackCap            = time.Hour * 24
 	DefaultStateWaitLookbackLimit = abi.ChainEpoch(20)
+	DefaultRateLimitTimeout       = time.Minute * 10
```

Comment:
Does this mean a hanging request for 10 minutes?

Reply:
We would override this.

Reply:
I'd set this to some reasonably high value; 5s is probably good enough